### PR TITLE
Fix eor sky model

### PR DESCRIPTION
**Pull request by steven-murray**
This adds back the EOR sky model creation routine, with the fix that only a constant monopole is added to the raw data.

### CODE REVIEW

File: core/sky_model.py [hunk 1]
```diff
@@ -73,48 +74,59 @@ def dnds_franzen(s, a=None, norm=False):
...
+
+    The model file is here assumed to contain Nfreqs healpix maps. The format of the
+    file is custom, as defined by @zacharymartinot, and the sky map is created by
+    his redshifted_gaussian_field code.
+    """
+    model_dir = utils.SKYDIR / "eor"
+
+    with h5py.File(model_dir / model_file, "r") as fl:
+        nside = int(fl["nside"][()])
+        freqs = fl["frequencies_mhz"][:] * 1e6 << units.Hz  # units Hz
```
Comment (piyanatk) [inline core/sky_model.py:88]: Why bit shifting with the unit?

File: core/sky_model.py [hunk 2]
```diff
@@ -73,48 +74,59 @@ def dnds_franzen(s, a=None, norm=False):
...
+        # HEALPix array -- dimension (nfreqs, npix) -- unit Jy/sr
+        # We must read in the whole thing to set the monopole.
+        hmaps = fl["healpix_maps_Jy_per_sr"][:, :]
+
+    hmaps <<= units.Jy / units.sr
+
+    npix = hp.nside2npix(nside)
+
+    # Shift the maps values so there are no negative values
+    # Note: we move the WHOLE sky up by a set amount so that the minimum value is 1.0.
```
Comment (piyanatk) [inline core/sky_model.py:99]: "WHOLE sky at ALL frequencies" -- Could be a bit more specific.

**Review by piyanatk — APPROVED** [review]
Look good Steven. Just a one question on the shifting `<<` operator.

File: core/sky_model.py (hunk 1 above)
Comment (steven-murray) [reply core/sky_model.py:88]: This is recommended by https://docs.astropy.org/en/stable/units/quantity.html#creating-and-converting-quantities-without-copies to not make an unnecessary copy of the data.

File: core/sky_model.py (hunk 2 above)
Comment (steven-murray) [reply core/sky_model.py]: Good call, fixed.

**Pull request merged**